lib/cmps/sourcepoint-frame.ts

```diff
@@ -78,12 +78,12 @@ export default class SourcePoint extends AutoConsentCMPBase {
       // toggles with 2 buttons
       const toggles = document.querySelectorAll('.priv-purpose-container .sp-switch-arrow-block a.neutral.on .right') as NodeListOf<HTMLElement>;
       for (const t of toggles) {
-        click([t]);
+        t.click()
```

Comment:
Shall we keep this? I think it's handy to use the same rule-executor functions like `click()` in code-based rules, so they emit the same logs

Reply:
I found supporting `ElementSelector | HTMLElement` in the `click` method signature a bit problematic, so I'd rather avoid it.